packages/help-center/src/hooks/use-is-wapuu-enabled.ts

```diff
@@ -25,5 +27,5 @@ export const useIsWapuuEnabled = () => {
 		return false;
 	}
 
-	return isWapuuConfigEnabled && ! isSiteJetpack;
+	return ( wapuuAssistantEnabled || isWapuuConfigEnabled ) && ! isSiteJetpack;
```

Comment:
Can the experiment be configured in the backend with those conditionals? I've never set up anything like that. Otherwise, I don't know how to avoid the rest of conditions